docs/reference/method/MongoDBModelIndexInfo-is2dSphere.txt

```diff
@@ -0,0 +1,58 @@
+==========================================
+MongoDB\\Model\\IndexInfo::is2dSphere()
+==========================================
```

Comment:
IIRC, RST emits a warning if the character length of headings and lines are not equal. Three characters should be removed on this and the first line.